src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,29 @@
+.navbar {
+    background-color: var(--white);
+    color: var(--black);
+}
+
+.greytext {
+    color: var(--blue);
+}
+
+.redtext {
+    color: var(--red);
+}
```

Comment:
you may name them as `grey-text` and `red-text` for better readability

Reply:
sure